### PR TITLE
Fix typos in README and bitcoin spelling

### PR DESCRIPTION
**Pull request by PulpCattel**
Some typos fixing.

Also makes the spelling of `B/bitcoin` consistent (`Bitcoin` is the protocol, `bitcoin` is the asset, no plural).

I'm uncertain if/how to fix this paragraph:

```
How CoinSwap works

A two-party coinswap are where Alice and Bob swap a coin in a non-custodial way, where neither party can steal from each other. At worst, they can waste time and miner fees.
```

In particular, `A two-party coinswap are where Alice and Bob swap a coin ...` does not seem right to me.

### CODE REVIEW

**Comment by MaxHillebrand** [comment]
`In a two-party coinswap, Alice and Bob can swap a coin in a non-custodial way, where neither party can steal from each other. At worst, they can waste time and miner fees.`

**Review by MaxHillebrand — APPROVED** [review]
ACK b540b0a, all looks good.

**Comment by PulpCattel** [comment]
I like @MaxHillebrand version.

File: README.md [hunk 1]
```diff
@@ -124,13 +124,13 @@ total balance = 0.14974828 BTC
 
 ### How CoinSwap works
 
-A two-party coinswap are where Alice and Bob swap a coin in a non-custodial where neither party can steal from each other. At worst they can waste time and miner fees.
+A two-party coinswap are where Alice and Bob swap a coin in a non-custodial way, where neither party can steal from each other. At worst, they can waste time and miner fees.
```
Comment (MaxHillebrand) [inline README.md:127]:
```suggestion
In a two-party coinswap, Alice and Bob can swap a coin in a non-custodial way, where neither party can steal from each other. At worst, they can waste time and miner fees.
```

**Comment by PulpCattel** [comment]
@chris-belcher are you okay too with Max's suggestion? Is there something you want me to change before I squash?

**Comment by chris-belcher** [comment]
Yes yes. Thanks for putting up with my lack of spellcheck/proofreading

File: README.md (hunk 1 above)
Comment (PulpCattel) [reply README.md]: Done in ea022c7

**Review by MaxHillebrand — APPROVED** [review]
ACK ea022c7

all reasonable & trivial, no unwanted changes.

Thanks @PulpCattel! :black_heart:

**Pull request merged**